sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -85,6 +85,27 @@ class CodegenContext {
   val references: mutable.ArrayBuffer[Any] = new mutable.ArrayBuffer[Any]()
 
   /**
+   * Add an object to `references`.
+   *
+   * Returns the code to access it.
+   */
+  def addReferenceObj(obj: Any): String = {
+    addReferenceObj(obj, null)
+  }
+
+  /**
+   * Add an object to `references`.
+   *
+   * Returns the code to access it.
+   */
+  def addReferenceObj(obj: Any, className: String): String = {
```

Comment:
Yea but looks like it is always null. Why not just remove this and move the body into the previous function?